src/commons/modifiers.ts

```diff
@@ -322,9 +334,10 @@ export function extractComponentProps(component: any, props: Dictionary<any>, ig
 }
 
 //@ts-ignore
-export function getThemeProps(props = this.props, context = this.context) {
-  //@ts-ignore
-  const componentName = this.displayName || this.constructor.displayName || this.constructor.name;
+export function getThemeProps(props = this.props, context = this.context, componentDisplayName = '') {
```

Comment:
I know it's not related to this PR, but what was the idea of putting this function here? is it related to the modifiers somehow?